Makefile

```diff
@@ -114,19 +115,23 @@ utility: engine-dependencies engine
 
 core:
 	@command -v $(MSBUILD) >/dev/null || (echo "OpenRA requires the '$(MSBUILD)' tool provided by Mono >= 5.4."; exit 1)
+ifneq ("$(MOD_SOLUTION_FILES)","")
 	@find . -maxdepth 1 -name '*.sln' -exec $(MSBUILD) -t:restore \;
-ifeq ($(WIN32), $(filter $(WIN32),true yes y on 1))
-	@find . -maxdepth 1 -name '*.sln' -exec $(MSBUILD) -t:build -p:Configuration="Release-x86" \;
-else
-	@$(MSBUILD) -t:build -p:Configuration=Release
-	@find . -maxdepth 1 -name '*.sln' -exec $(MSBUILD) -t:build -p:Configuration=Release \;
+	ifeq ($(WIN32), $(filter $(WIN32),true yes y on 1))
```

Comment:
Makefile syntax defines that any line starting with a tab is passed to `/bin/sh` for processing.
Remove the tab here and on the `else` and `endif` to fix the compile errors on macOS.

Reply:
Looks a bit stupid now, but should be syntactically correct. Both variants worked on Linux.